tests/cursor/cursor-tailable-001.phpt

```diff
@@ -22,7 +22,7 @@ function insert(MongoDB\Driver\Manager $manager, $from, $to = null)
 
     $writeResult = $manager->executeBulkWrite(NS, $bulkWrite);
 
-    printf("Inserted %d document(s): %s\n", $writeResult->getInsertedCount(), implode(range($from, $to), ', '));
+    printf("Inserted %d document(s): %s\n", $writeResult->getInsertedCount(), implode(', ', range($from, $to)));
```

Comment:
I was curious as to how this ever worked, but it appears to be noted on https://www.php.net/manual/en/function.implode.php

Did PHP 7.4 actually fix/deprecate that?

Reply:
Yes, this is deprecated in 7.4, along with some similar „features“ in other functions.